### DISCUSSION TITLE
MASS LOSS RATE

### DISCUSSION THREAD

**Discussion by SaharRasoulpour** [firemodels/fds#13412] [General]
Hello, I would appreciate your help in solving my problem. According to the discussion in #13130, I resolved the maximum temperature fluctuation on the left-hand side of my domain for a pine bed, based on the kinetic parameters. Since I don't have values for char oxidation, I used the values from vegetation.txt in FDS. My temperature plot gives the desired results, but when I look at the mass change over time plot between 50 and 60 seconds, there is a sharp decrease in mass that causes an undesirable increase in the mass loss rate (MLR) in my MLR over time plot. Could this be due to the char kinetic parameters, or could it be related to other parameters? Your response would be greatly appreciated.

**Comment by ericvmueller** [comment]
It is possible this is related to the char model as this is an ongoing topic of research. The parameters are suggested values but may not be appropriate for your conditions or materials. However, there are also plenty of other possibilities. Can you post your most recent input file here?

**Reply by SaharRasoulpour** [reply]
Yes, here you can find my FDS input files

[sample.txt](https://github.com/user-attachments/files/16950502/sample.txt)
[vegetation_model.txt](https://github.com/user-attachments/files/16950503/vegetation_model.txt)
